Comp2023-Imported/src/main/java/frc/robot/subsystems/Swerve.java

```diff
@@ -503,6 +526,10 @@ public Pose2d getPose( )
 
   public void resetOdometry(Pose2d pose)
   {
+    DataLogManager.log("heading " + m_heading);
+    DataLogManager.log("positions " + getPositions( ));
+    DataLogManager.log("pose " + pose);
+
```

Comment:
I think we can delete this since it was used for testing and will be putting logs every time "resetOdometry" is called (so very often).

Reply:
I deleted my DataLogManager command for the raw pose but these are the ones we used before

Reply:
resetOdometry shouldn't be called very often (f we're doing things right), but those log messages probably won't be needed in the long term. Ok to leave in for now if it helps.